packages/util/package.json

```diff
@@ -63,7 +63,8 @@
   },
   "dependencies": {
     "lodash": "^4.17.21",
-    "serialize-error": "^8"
+    "serialize-error": "^8",
+    "type-fest": "^2.19.0"
```

Comment:
Shouldn't this be in `devDependencies`? Seems like it only has types

Reply:
I'm a bit confused too. Yes, it has only types.
At the same time, would a user of the `util` library, importing DeepPartial, be able to see what `Primitive` type is? (since it comes from type-fest)

Reply:
Good question, don't know, could you please try and see how it works?

Reply:
I have tried on a test project and it shows the types correctly.
[cb6c0bc](https://github.com/input-output-hk/cardano-js-sdk/pull/396/commits/cb6c0bc3fbf52e3131186a505b45f2d23879a324)